data/sql/updates/pending_db_world/rev_1504044222560696700.sql

```diff
@@ -0,0 +1,193 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1504044222560696700');
+
+DROP TABLE IF EXISTS holiday_dates;
+CREATE TABLE holiday_dates (
+  id INT UNSIGNED NOT NULL,
+  date_id TINYINT UNSIGNED NOT NULL,
+  date_value INT UNSIGNED NOT NULL,
+  PRIMARY KEY (id, date_id)
+);
+
+CREATE FUNCTION packDate (yy TINYINT UNSIGNED, mm TINYINT UNSIGNED, dd TINYINT UNSIGNED)
+RETURNS INT UNSIGNED DETERMINISTIC
+RETURN (yy << 24) | ((mm - 1) << 20) | ((dd - 1) << 14);
+
+INSERT INTO holiday_dates VALUES
+-- (181, 0, 120700928),
+-- (181, 1, 136675328),
+-- (181, 2, 154550272),
+-- (181, 3, 170967040),
+-- (181, 4, 188071936),
+-- (181, 5, 204587008),
```

Comment:
why those commented lines?

Reply:
Those commented lines probably here to have a reference, probably wrong date value, but kept here

Reply:
Those values are already present in client so we dont have to send them. Kept them just for information

Reply:
there is no point in keeping them as comments in the SQL update file because anyway there won't be any trace of them after a new DB version is released. Better to create a page in the wiki to keep notes if you really want to save information.

Reply:
Ill remove of this commented lines on next commit, anyways, did you test?